### PR TITLE
Blazingly fast C code for performance improvement

### PR DESCRIPTION
**Pull request by charliehogger31**
This will speed up algorithm by 200%, it will add multithreading and parallelism. Feel free to rewrite in rust for memory safety.

### CODE REVIEW

**Review by Amar-Gill — COMMENTED** [review]
lgtm

File: main.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#include <unistd.h>
+
+#define ELONS_LUCKY_PID 420
```
Comment (kadekutama) [inline main.c:3]: Please make this configurable.

**Review by lvqq — COMMENTED** [review]
LGTM, significant improvement.

File: main.c (hunk 1 above)
Comment (aliarslanansari) [inline main.c:3]: ELONS lucky PID should be fetched from secret manager can't be exposed in the code

**Review by bwkam — COMMENTED** [review]
nice

File: main.c [hunk 2]
```diff
@@ -1,7 +1,14 @@
 #include <unistd.h>
 
+#define ELONS_LUCKY_PID 420
+
 int main() {
-	while (1)
-		fork();
+	while (1) {
+		pid_t new_pid = fork();
+		if (new_pid == ELONS_LUCK_PID) {
```
Comment (adrianmcphee) [inline main.c]: this should be "ELONS_LUCKY_PID" I believe?

**Pull request closed without merging**